### PR TITLE
[INSIGHTS-4] Add Insights user-facing documentation

### PR DESCRIPTION
**Pull request by dinoboy197**
Adding Insights documentation to the dev center. Introductory language taken from [the Product Spec](https://docs.google.com/document/d/1QD9cKyjZmV4Lm1x4jJBnRg4wc9iuC3Il66Uz2FsSOkU/edit?disco=AAAAG5CCoWI). I wrote the remainder of the text. Note that there are some screenshots and code snippets that will be added in [a later piece of work](https://algorithmia.atlassian.net/browse/INSIGHTS-31) once the client library implementation and UX design implementation is finished; that will come before 9/22.

You can play around with it for real in our testing environment - https://insights1.enthalpy.click/developers

Here is the main Insights page, nestled under Algorithmia Enterprise:
 image
image
image
image
image
image

And here are the additions for the Python client library:
image

And R:
image

### CODE REVIEW

**Review by JamesHoover — APPROVED** [review]
This looks great, I wouldn't change a thing

File: _pages/algorithmia-enterprise/insights.md
Comment (eliperelman) [inline _pages/algorithmia-enterprise/insights.md]: Not sure if you wanted to change it for consistency, but there is a stylized apostrophe in the second `you'll`.

File: _pages/algorithmia-enterprise/insights.md
Comment (eliperelman) [inline _pages/algorithmia-enterprise/insights.md]: Replace "request id" with "request ID".
Replace "author / manager" with "author/manager" or "author or manager".

File: _pages/algorithmia-enterprise/insights.md
Comment (eliperelman) [inline _pages/algorithmia-enterprise/insights.md]: Insert comma after `cute_cats_detected`.

File: _pages/algorithmia-enterprise/insights.md
Comment (eliperelman) [inline _pages/algorithmia-enterprise/insights.md]: Replace "One" with "Once".
Double "that" usage with "that specific version of that algorithm". Could we change to "that specific version of the algorithm"?

File: _pages/clients/python.md
Comment (eliperelman) [inline _pages/clients/python.md]: Missing the closing paren for "(a feature of".

File: _pages/clients/python.md
Comment (eliperelman) [inline _pages/clients/python.md]: Replace "report" with "Report".

File: _pages/clients/r.md
Comment (eliperelman) [inline _pages/clients/r.md]: Missing the closing paren for "(a feature of".

File: _pages/clients/r.md
Comment (eliperelman) [inline _pages/clients/r.md]: Replace "report" with "Report".

File: _pages/clients/r.md
Comment (eliperelman) [inline _pages/clients/r.md]: Replace `$` with `.`.

**Review by eliperelman — COMMENTED** [review]
LGTM, just have some grammar nits.

File: _pages/algorithmia-enterprise/insights.md
Comment (JamesHoover) [inline _pages/algorithmia-enterprise/insights.md]: I am calling the page 'Algorithm Insights' in the UI to be clear it's not platform insights.

Comment (ZachEddy) [inline _pages/algorithmia-enterprise/insights.md]: > _Insights_ ... "System Actions"

When describing which pages to click through, does it make sense to use "quotes" or _italics_ but not both?

Comment (dinoboy197) [reply _pages/algorithmia-enterprise/insights.md]: OK. Is platform insights documented in the dev center so I can read and educate myself about it?

**Review by ZachEddy — APPROVED** [review]

File: _pages/algorithmia-enterprise/insights.md
Comment (JamesHoover) [reply _pages/algorithmia-enterprise/insights.md]: No, not specifically just want the tab to be clear what you're clicking on

Comment (dinoboy197) [reply _pages/algorithmia-enterprise/insights.md]: It probably does, though I was following other pages that follow this convention. Having a "style guide" for the dev center would probably help out here for contributors. @algorithmiaio/ux food for thought

Comment (dinoboy197) [reply _pages/algorithmia-enterprise/insights.md]: Ok, I'll change this wording here. Thanks!

File: _pages/clients/r.md
Comment (dinoboy197) [reply _pages/clients/r.md]: Other method calls documented in this file use `$` as the object / method separator. Should this deviate from those examples?

Comment (eliperelman) [reply _pages/clients/r.md]: Oh my bad, thought this was a typo. :joy:

File: _pages/algorithmia-enterprise/insights.md
Comment (JamesHoover) [reply _pages/algorithmia-enterprise/insights.md]: Yes, let's use quotes. Here is the style guide: https://docs.google.com/document/d/1PPVfgMkX7-EVGLPMhN1E485CAXu9QfhSLKM0lZhnZdU/edit?usp=sharing

This case is not covered so we'll add it

Comment (dinoboy197) [reply _pages/algorithmia-enterprise/insights.md]: Excellent! May I make a recommendation? Update the [Contributing](https://github.com/algorithmiaio/dev-center#contributing) section of the README file in this repository to link to that style guide. Without that reference, there's no way other Algorithmia engineers are going to know that file exists or remember what the link is :)

I'm aware that this repository is open for public contribution, but the practical reality is that the only people who ever edit this repository are Algorithmia engineers :)

**Review by JamesHoover — APPROVED** [review]
🎉

**Comment by dinoboy197** [comment]
@StephanieKim I need final approval from you on this to be able to ship. You can see the finished product live at https://insights1.enthalpy.click/developers. Note that there are some screenshots and code snippets that will be added in a later piece of work once the client library implementation and UX design implementation is finished; that will come before 9/22.

**Review by ZachEddy — APPROVED** [review]
🚢

**Review by stevielkim — CHANGES_REQUESTED** [review]
This looks amazing! Thanks for doing such a great job. I only have one change needed but it needs to happen throughout the guide, but wherever it says "Insights" can we change to "Algorithmia Insights" for branding purposes?

**Review by eliperelman — APPROVED** [review]

**Comment by dinoboy197** [comment]
> I only have one change needed but it needs to happen throughout the guide, but wherever it says "Insights" can we change to "Algorithmia Insights" for branding purposes?

Sure thing - I had that was implicit since we're Algorithmia :) But I realize that maybe marketing is working off of something like "Check out Algorithmia's new product feature, Algorithmia Insights!" Making the change now.

**Comment by dinoboy197** [comment]
> Sure thing - I had that was implicit since we're Algorithmia :) But I realize that maybe marketing is working off of something like "Check out Algorithmia's new product feature, Algorithmia Insights!" Making the change now.

OK @StephanieKim changes added and deployed to https://insights1.enthalpy.click/developers. Please take a look and let me know if you'd like any other changes, or approve if not!

**Pull request closed without merging**
